source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -71,7 +71,74 @@ browser as shown in figure 1.
     Confirmation the Apache service is running.
 
 The :file:`index.html` file is located in the :file:`/usr/share/httpd/htdocs`
-directory of your host system.
+directory of your host system.  We will copy this file into a new location 
+after we modify the configuration.
+
+Change the Default Configuration and Data Directory
+===================================================
+
+|CL| is designed to be a stateless operating system which means that you will
+need to create an optional configuration file.  The default location of the
+Apache configuration file, httpd.conf, is located in the
+/etc/share/defaults/httpd directory, which can be overwritten as part of the
+stateless paradigm.  This default .conf file includes the following directives
+that allow for additional locations of configuration definitions:
+
+.. code-block:: console
+
+   # Virtual hosts
+   IncludeOptional /usr/share/defaults/httpd/conf.d/*.conf
+   IncludeOptional /usr/share/defaults/httpd/conf.modules.d/*.conf
+   IncludeOptional /etc/httpd/conf.d/*.conf
+   IncludeOptional /etc/httpd/conf.modules.d/*.conf
+
+
+For this tutorial, we will create the directory structure for /etc/httpd/conf.d
+and then create the httpd.conf file within /etc/httpd/conf.d directory and
+include the variable DocumentRoot.
```

Comment:
Use the ``:file:`` role for the files and directories in this paragraph. Use double backticks for all literal content when no role exists:
``DocumentRoot``.